vendor/github.com/sirupsen/logrus/README.md

```diff
@@ -1,8 +1,28 @@
-# Logrus <img src="http://i.imgur.com/hTeVwmJ.png" width="40" height="40" alt=":walrus:" class="emoji" title=":walrus:"/>&nbsp;[![Build Status](https://travis-ci.org/sirupsen/logrus.svg?branch=master)](https://travis-ci.org/sirupsen/logrus)&nbsp;[![GoDoc](https://godoc.org/github.com/sirupsen/logrus?status.svg)](https://godoc.org/github.com/sirupsen/logrus)
+# Logrus <img src="http://i.imgur.com/hTeVwmJ.png" width="40" height="40" alt=":walrus:" class="emoji" title=":walrus:"/> [![Build Status](https://travis-ci.org/sirupsen/logrus.svg?branch=master)](https://travis-ci.org/sirupsen/logrus) [![GoDoc](https://godoc.org/github.com/sirupsen/logrus?status.svg)](https://godoc.org/github.com/sirupsen/logrus)
 
 Logrus is a structured logger for Go (golang), completely API compatible with
 the standard library logger.
 
+**Logrus is in maintenance-mode.** We will not be introducing new features. It's
```

Comment:
FYI: Highlighting this one (https://github.com/sirupsen/logrus/commit/4ecd9a62bd57b6533e0d87ae23a6b10adc32b5ce)

Reply:
Kudos to @sirupsen Simon!

Reply:
Yeah, you know, except, I plan to change the casing at some point. For fun.

🤦

Reply:
LOL change your handle to "aka-Sirupsen"

But yes, Kudos! Logrus is definitely still serving us well (I don't think we're looking for new features, so not sure if there's a reason for us to move away to something else immediately)

Reply:
Only reason would be performance. Hands are tied in Logrus on that one.

Reply:
Ooof @thaJeztah :)